docker-vertica/docker-entrypoint.sh

```diff
@@ -10,12 +10,13 @@ start_cron(){
 # because we have a Persistent Volume that backs /opt/vertica/config, so
 # it starts up empty and must be populated
 copy_config_files() {
-    cp -r /home/dbadmin/logrotate/* /opt/vertica/config/
+    mkdir -p /opt/vertica/config/licensing
+
+    mv /home/dbadmin/logrotate/* /opt/vertica/config/ 2>/dev/null || true
     rm -rf /home/dbadmin/logrotate
```

Comment:
Matt, why is this directory getting removed?  It looks like the dockerfile is copying a couple files (directory?) and then this entrypoint is copying them back.  I think everything I changed is safe, but I wanted to remove this rm command and wasn't sure about it.

Reply:
It is fine to skip this.  I think we only removed it because the files were copied to /opt.  It might be confusing to have the files in two places.